lib/validates_zipcode/cldr_regex_collection.rb

```diff
@@ -132,123 +134,3686 @@ module CldrRegexpCollection
+          '42511',
+          'AZ42511',
+          'XY12Z',
+          'A12',
+          '123',
+          'ABC'
+        ],
+        invalid: [
+          nil,
+          '',
+          'invalid_zip',
+          'AB',
+          '12'
+        ]
+      },
+      UY: {
+        valid: [
+          '42511',
+          'AZ42511',
+          'XY12Z',
+          'A12',
+          '123',
+          'ABC'
+        ],
+        invalid: [
+          nil,
+          '',
+          'invalid_zip',
+          'AB',
+          '12'
+        ]
+      },
+      VC: {
+        valid: [
+          'VC2181',
+          'VC6376',
+          'VC4702'
+        ],
+        invalid: [
+          nil,
+          '',
+          'invalid_zip',
+          'VC281',
+          'VC63726',
+          'CV4702'
+        ]
+      },
+      VI: {
+        valid: [
+          '07113',
+          '02566',
+          '66261'
+        ],
+        invalid: [
+          nil,
+          '',
+          'invalid_zip',
+          '1234',
+          '12345-12345',
+          'D0D0D0'
+        ]
+      },
+      VG: {
+        valid: [
+          '42511',
+          'AZ42511',
+          'XY12Z',
+          'A12',
+          '123',
+          'ABC'
+        ],
+        invalid: [
+          nil,
+          '',
+          'invalid_zip',
+          'AB',
+          '12'
+        ]
+      },
+      VN: {
+        valid: [
+          '071134',
+          '025663',
+          '662612',
+          '071135'
+        ],
+        invalid: [
+          nil,
+          '',
+          'invalid_zip',
+          '12345',
+          '123456-123456',
+          'D0D0D0'
+        ]
+      },
+      VU: {
+        valid: [
+          '42511',
+          'AZ42511',
+          'XY12Z',
+          'A12',
+          '123',
+          'ABC'
+        ],
+        invalid: [
+          nil,
+          '',
+          'invalid_zip',
+          'AB',
+          '12'
+        ]
+      },
+      WS: {
+        valid: [
+          '42511',
+          'AZ42511',
+          'XY12Z',
+          'A12',
+          '123',
+          'ABC'
+        ],
+        invalid: [
+          nil,
+          '',
+          'invalid_zip',
+          'AB',
+          '12'
+        ]
+      },
+      YE: {
+        valid: [
+          '42511',
+          'AZ42511',
+          'XY12Z',
+          'A12',
+          '123',
+          'ABC'
+        ],
+        invalid: [
+          nil,
+          '',
+          'invalid_zip',
+          'AB',
+          '12'
+        ]
+      },
+      ZW: {
+        valid: [
+          '42511',
+          'AZ42511',
+          'XY12Z',
+          'A12',
+          '123',
+          'ABC'
+        ],
+        invalid: [
+          nil,
+          '',
+          'invalid_zip',
+          'AB',
+          '12'
+        ]
+      }
     }.freeze
 
     def regexp_for_country_alpha2(alpha2)
       alpha2 = alpha2.to_s.upcase.to_sym
 
       ZIPCODES_REGEX[alpha2]
     end
+
+    def values_for_country_alpha2(alpha2)
```

Comment:
Ok, now I see why you added the list here. 

In this case, and to keep the backwards compatibility for anyone using the gem (considering performance as part of the compatibility), I then suggest we split this feature into a different file that does not get required when the gem is required. 

We'd then document the optional usage of the feature by adding an extra `require .../generator`  and then calling `ValidatesZipcode::Generator.sample(alpha2)`